### DISCUSSION TITLE
GUI: simulation results visualization styling

### DISCUSSION THREAD

**Discussion by chenghuzi** [jonescompneurolab/hnn-core#510] [GSoC]
Now we're just using a very basic settings for visualization for all types of figures. If we want to allow the user to adjust some styling parameters, which way do you think is good?
* Add a dropdown menu for different plots
* A textarea input field for users to freely add any configurations they want and we just pass the content to `kwargs` in those functions.

Also, we may need to consider Mainak's prototype https://github.com/jonescompneurolab/hnn-core/issues/497#issuecomment-1181258384, which seems to make things easier.

**Comment by jasmainak** [comment]
I think it's okay for the configurations to be basic ... maybe for some plots we might need some options but let's get the GUI in the hands of users and run the tutorials to see what are the blockers. Maybe one day before this week's meeting, you can assign me, Ryan, and Nick one tutorial to run through with the GUI?
